Qase_Test/Src/Pages/Project/ProjectSettingsPage.cs

```diff
@@ -1,31 +1,29 @@
 using OpenQA.Selenium;
 using Qase_Test.Core.Browser.Service;
 using Qase_Test.Pages.Base;
-using Qase_Test.Utils;
 using Qase_Test.Wrappers;
 
 namespace Qase_Test.Pages.Project
 {
     public class ProjectSettingsPage : BasePage
     {
-        private readonly By _deleteProjectButtonSelector = By.CssSelector(".btn-cancel");
         private const string ProjectDescriptionLabel = "Description";
-        private static readonly By ProjectSettingsTitleText = By.XPath($"//h1[text()='Settings']");
+        private readonly By _deleteProjectButtonSelector = By.CssSelector(".btn-cancel");
 
-        public ProjectSettingsPage() : base(ProjectSettingsTitleText)
+        public ProjectSettingsPage() : base(By.XPath($"//h1[text()='Settings']"))
         {
         }
 
         public void ClickDeleteProject()
         {
             try
             {
-                BrowsersService.GetDriver.SwitchTo().Alert().Accept();
+                BrowsersService.Driver.SwitchTo().Alert().Accept();
             }
             catch (NoAlertPresentException ex)
             {
                 Logger.Error(ex.Message);
-                WebElementActions.GetElement(_deleteProjectButtonSelector).Click();
+                BaseElement.GetElement(_deleteProjectButtonSelector).Click();
```

Comment:
we talked about such cases, you should create instance for Input, Button or Dropdown